### ISSUE TITLE
[New Primitive] `LiveCursor`

### ISSUE THREAD

**Issue by jjenzz** [radix-ui/primitives#1323]
I'm not sure what the primitive would be called but `LiveCursor` paints a more accurate picture in my mind of the idea that originated here https://github.com/radix-ui/primitives/discussions/1090.

Obviously this pattern is likely difficult to make readable by SRs and may even be an a11y failure so perhaps it can follow a similar approach to `HoverCard` in that it would be for sighted users only.

Examples in the wild:

- Figma multiplayer mode
- [Liveblocks live cursor](https://liveblocks.io/docs/examples/live-cursors) 
- Vercel conference marketing website multiplayer (can't find a link for this now but [here is a video showing the gist](https://youtu.be/ze8ycxc1UzE?t=332))

**Comment by lewisvrobinson** [comment]
Chris from Liveblocks recently posted a great article on animating live cursors and dealing with latency. 

https://liveblocks.io/blog/how-to-animate-multiplayer-cursors

Could come in handy!

**Comment by franklinjavier** [comment]
https://github.com/radix-ui/primitives/discussions/1090#discussioncomment-6620771
